config/testing.go

```diff
@@ -34,24 +38,32 @@ Filtering:
 
 // LoadTestingConfig loads the hard coded testing config
 func LoadTestingConfig(mongoURI string) (*Config, error) {
-	Version = "v0.0.0+testing"
-	ExactVersion = "v0.0.0+testing"
-	var config = new(Config)
-	static, err := parseStaticConfig([]byte(testConfig))
-	if err != nil {
-		return config, err
+	config := &Config{}
+
+	// Initialize table config to the default values
+	if err := defaults.Set(&config.T); err != nil {
+		return nil, err
+	}
+
+	// Initialize static config to the default values
+	if err := defaults.Set(&config.S); err != nil {
+		return nil, err
 	}
-	config.S = *static
 
 	config.S.MongoDB.ConnectionString = mongoURI
 
-	config.T = *loadTableConfig()
+	// Deserialize the yaml file contents into the static config
+	if err := parseStaticConfig([]byte(testConfig), &config.S); err != nil {
+		return nil, err
+	}
+
+	config.S.Version = "v0.0.0+testing"
```

Comment:
It was a little confusing before how `Version` and `ExactVersion` variables were used. They were setting variables in another scope which then led to this happening anyway.  I feel this is a little more explicit.